bedrock/settings/base.py

```diff
@@ -234,7 +234,7 @@ def lazy_langs():
 FEED_CACHE = 3900
 DOTLANG_CACHE = 600
 
-DOTLANG_FILES = ['main', 'download_button']
+DOTLANG_FILES = ['main', 'download_button', 'navigation']
```

Comment:
Can we try setting this to `'navigation', 'download_button', 'main'`?

main.lang has a ton of strings, my hope is that we use the ones in navigation first like this